classification/models/classification.py

```diff
@@ -1086,6 +1086,10 @@ def process_entry(self, cell: VCDataCell, source: str):
             value = value.replace('&gt;', '>').replace('&lt;', '<')
 
         if value is None and e_key.mandatory and not e_key.exclude_namespace:
+            if e_key.key == SpecialEKeys.CONDITION:
+                # Gene condition is not mandatory if we have a condition resolution
+                if self.condition_resolution_obj:
+                    return None
```

Comment:
rather than `return None` just have this scenario not raise the mandatory validation, but still run through the rest of process_entry as there's a bit more that code does.